sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/models/ShareFileHttpHeaders.java

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.file.share.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.implementation.util.ImplUtils;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+/**
+ * Additional parameters for a set of operations, such as: Files_create,
+ * Files_setHTTPHeaders.
+ */
+@JacksonXmlRootElement(localName = "share-file-http-headers")
```

Comment:
Should this be generating into `implementation`? I thought the previously named class was just in models, also does `FileHttpHeaders` need to be deleted and replaced with this?

Reply:
FileHttpHeaders.java has deleted. It is originally under model as it is a return type for public API. Forgot to delete the one under impl.